api/src/main/java/org/pmiops/workbench/notebooks/LeonardoNotebooksClient.java

```diff
@@ -25,9 +25,6 @@ Cluster createCluster(String googleProject, String clusterName, String workspace
   /** Deletes a notebook cluster */
   void deleteCluster(String googleProject, String clusterName) throws WorkbenchException;
 
-  /** Lists all existing clusters */
-  List<Cluster> listClusters(String labels, boolean includeDeleted) throws WorkbenchException;
```

Comment:
this was removed because it was not used, and it's easy to confuse this with ClusterController.listClusters()